### ISSUE TITLE
Use alternative 'upstream' for driver bindings

### ISSUE THREAD

**Issue by gbraad** [crc-org/vfkit#10]
It's suggested to use htps://github.com/machine-drivers/vz as the upstream for the bindings.

Ref:

  * https://github.com/code-ready/crc/issues/3362

Perhaps it might even be beneficial if parts of this codebase becomes part of the machine-drivers codebase?

**Comment by afbjorklund** [comment]
I don't think that the "machine-drivers" organization is a good place to host upstream dependencies.

* https://github.com/machine-drivers/vz/issues/1

I think that the fork should be archived, at least until a docker-machine-driver-vfkit exists...

And it is unlikely to do so, since there is/was no support for arm64 in the boot2docker iso.

**Comment by afbjorklund** [comment]
You could of course use crc-org/vz, if you need to fork... But machine-drivers is retiring soon.

Since it (docker/machine) has not been updated since 2019, it probably will see EOL in 2024.
